cmd/gitsign-credential-cache/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"path/filepath"
+
+	"github.com/sigstore/gitsign/internal/cache"
+	cachepb "github.com/sigstore/gitsign/internal/cache/cache_go_proto"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	home, err := os.UserHomeDir()
```

Comment:
(not blocking) Why do I see people using https://github.com/mitchellh/go-homedir when this is in the stdlib? Is go-homedir better in some way? 🤔

Reply:
os.UserHomeDir() was added in Go 1.12 (~May 2019). Where as go-homedir was created in 2015.